src/common/snippets/src/op/subgraph.cpp

```diff
@@ -256,16 +256,11 @@ auto Subgraph::wrap_node_as_subgraph(const std::shared_ptr<ov::Node>& node) -> s
 }
 
 void Subgraph::fill_empty_output_names(const Output<Node>& target_output_node, const Output<Node>& replacement_output_node) {
-    OPENVINO_SUPPRESS_DEPRECATED_START
     auto& out_tensor = target_output_node.get_tensor();
     const std::string new_name = ov::op::util::get_ie_output_name(replacement_output_node);
-    if (ov::descriptor::get_ov_tensor_legacy_name(out_tensor).empty()) {
-        ov::descriptor::set_ov_tensor_legacy_name(out_tensor, new_name);
-    }
```

Comment:
@ilya-lavrenov  @vurusovs  this changes will create core dump for test in following, any suggestion?
![image](https://github.com/openvinotoolkit/openvino/assets/52686861/ab1d10b9-a910-4d79-acc3-809b957805d8)

Reply:
Did you have any more information? I'm not sure legacy_name may cause such issue (looks like failure happened during `find_port(...)`)

Reply:
The error happened at here, the name is "output", but the map name is "output.1".
https://github.com/openvinotoolkit/openvino/blob/0ea791e9b7c2748768cbd9cf77b60c634443c19a/src/plugins/intel_cpu/src/infer_request.cpp#L353
![image](https://github.com/openvinotoolkit/openvino/assets/52686861/685b80a2-62fe-4b1e-9de7-374b24b5392c)